core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2222,7 +2222,7 @@ private[spark] object Utils extends Logging {
    */
   def portMaxRetries(conf: SparkConf): Int = {
     val maxRetries = conf.getOption("spark.port.maxRetries").map(_.toInt)
-    if (conf.contains("spark.testing")) {
+    if (isTesting || conf.contains("spark.testing")) {
```

Comment:
shall we just call `isTesting` here?

Reply:
Sorry, may I have this understanding of one-sided point. It is not just in the test call. But when we need to get the default value for spark.port.maxRetries is 100.  still need to set the `'spark.testing` .
Or in the Spark unit test set test sign. so I added it to here. thanks.